lib/globus_client/endpoint.rb

```diff
@@ -3,9 +3,9 @@
 class GlobusClient
   # The namespace for endpoint API operations
   class Endpoint
-    PATH_SEPARATOR = "/"
+    PATH_SEPARATOR = '/'
 
-    FileInfo = Struct.new(:name, :size)
+    FileInfo = Struct.new(:name, :size) # rubocop:disable Lint/StructNewOverride
```

Comment:
I wonder if renaming `:size` to `:file_size` would also get around this rubocop exlusion, which seems like a decent one.